android/src/main/java/com/amplitude/android/Configuration.kt

```diff
@@ -76,4 +77,39 @@ open class Configuration @JvmOverloads constructor(
     companion object {
         const val MIN_TIME_BETWEEN_SESSIONS_MILLIS: Long = 300000
     }
+
+    @Deprecated("Please use 'autocapture.sessions' instead.", ReplaceWith("autocapture.sessions"))
+    var trackingSessionEvents: Boolean
+        get() = AutocaptureOption.SESSIONS in autocapture
+        set(value) {
+            if (value) autocapture += AutocaptureOption.SESSIONS else autocapture -= AutocaptureOption.SESSIONS
+        }
```

Comment:
Does this snippet of code disable session tracking with the following configuration?
```
val amplitude = Amplitude(
  Configuration(
    apiKey = AMPLITUDE_API_KEY,
    context = applicationContext,
    trackingSessionEvents = false,
    autocapture = true,
  )
)
```

IMO, the new config `autocapture` should take precedence over old configs `trackingSessionEvents` and `defaultTracking`.

Reply:
If any of the `trackingSessionEvents`, `defaultTracking.session`, or `AutocaptureOptions.SESSIONS in autocapture` is false, this means that one of them is set to false manually. So I think we should disable sessions in that case.

Reply:
Close this and will go as we discussed here https://amplitude.slack.com/archives/C04HSDQA2VB/p1722893976133219?thread_ts=1722556642.263359&cid=C04HSDQA2VB